app/models/submission.rb

```diff
@@ -36,7 +36,7 @@ class Submission < ApplicationRecord
   belongs_to :manager, class_name: 'User'
   has_many :evaluator_submission_assignments, dependent: :destroy
   has_many :evaluators, through: :evaluator_submission_assignments, class_name: "User"
-  has_many :evaluations, dependent: :destroy
+  has_many :evaluations, through: :evaluator_submission_assignments, dependent: :destroy
```

Comment:
the `evaluations` table has the `submission_id` with an index on it (Evaluation model `belongs_to :submission`). we can use that more efficiently to query directly instead of going through another table.
```suggestion
  has_many :evaluations, dependent: :destroy
```